codegen/core/src/test/scala/com/lightbend/akkasls/codegen/PackagingSuite.scala

```diff
@@ -17,8 +17,10 @@
 package com.lightbend.akkasls.codegen
 
 class PackagingSuite extends munit.FunSuite {
+  private val testData = TestData()
```

Comment:
TestData became a class (instead of object) so that it can be initialized with defaults (especially the difference between javaOuterClassName for Java vs Scala)